### DISCUSSION TITLE
v0.3.0

### DISCUSSION THREAD

**Discussion by Skryvvara** [Skryvvara/FocusFrame#55] [Announcements]
## What's Changed
* chore: improve readme by @Skryvvara in https://github.com/Skryvvara/FocusFrame/pull/52
* feat: add config defaults by @Skryvvara in https://github.com/Skryvvara/FocusFrame/pull/53
* feat: make hotkey configurable by @Skryvvara in https://github.com/Skryvvara/FocusFrame/pull/54

**Full Changelog**: https://github.com/Skryvvara/FocusFrame/compare/v0.2.0...v0.3.0

<hr /><em>This discussion was created from the release <a href='https://github.com/Skryvvara/FocusFrame/releases/tag/v0.3.0'>v0.3.0</a>.</em>

**Comment by jjumbc** [comment]
the only version that works for me is 0.1, 0.3, 0.2, and 0.1.1 crash immediately

windows 11 23H2, 7600x, 7900xt

**Reply by Skryvvara** [reply]
Hi @jjumbc,

Thank you for bringing this to my attention. There's an oversight with how the new config works. It tries to save the config under `%APPDATA%/FocusFrame/config.toml` but fails as the `%APPDATA%/FocusFrame` directory doesn't exist. Creating it manually should fix the issue for now. I will fix this issue v0.3.1 later today.

**Reply by jjumbc** [reply]
Thanks! I still found 0.1 very useful for getting a game with a weird Windowed mode that doesn't let you drag or resize the window to cooperate (Monster Hunter Wilds, huge upcoming release). Great stuff!

**Reply by Skryvvara** [reply]
I'm happy that I could be helpful :D I have created a v0.3.1 that should fix this issue for everyone now.
